src/matlab2tikz.m

```diff
@@ -98,6 +98,9 @@ function matlab2tikz(varargin)
 %   interpret tick labels as TeX. MATLAB(R) doesn't do that by default.
 %   (default: false)
 %
+%   MATLAB2TIKZ('arrowHeadSizeFactor', FLOAT, ...) allows to resize the arrow heads
+%   in quiver plots by rescaling the arrow heads by a positive scalar. (default: 10)
```

Comment:
I disagree with you. This does not mix two options: if the user provides his own value, we should use that. Otherwise, we use our default. It may seem a bit counter-intuitive with the default value "10" instead of a "1" or "0" or something, but that allows for a more straightforward passing of the data (otherwise, it's another `if value <= 0, hsz = 10, end` or `hsz = hsz * 10` somewhere in the code: it always introduces the magic value "10" somewhere). My choice here is to just publicly announce that factor to the user (since some users may wonder why we write `10*\pgfplotspointmetatransformed/1000`, so I think it makes sense to just show it).

The same happens for "MaxChunkSize", there we also use a magic constant that is based on some quick and dirty calculations.

In any case: I think this parameter should not survive the 2.0 release, by then we should have an automatic way to determine it and get rid of the parameter altogether.